inference-engine/src/vpu/myriad_plugin/myriad_executable_network.cpp

```diff
@@ -186,6 +187,10 @@ InferenceEngine::Parameter ExecutableNetwork::GetMetric(const std::string &name)
 
 InferenceEngine::CNNNetwork ExecutableNetwork::GetExecGraphInfo() {
     auto perfInfo = _executor->getPerfTimeInfo(_graphDesc._graphHandle);
+    if (_graphDesc._name == importedNetworkName)
```

Comment:
Anyway, I haven't found justification, that `operator==` is overloaded to compare `std::string` and `char[]` in C++11.
Could you prove it, please?

Reply:
Well, found :)